docs/guide/collections/configuration-reference.md

```diff
@@ -44,6 +44,14 @@ The string that is used when a UI needs a title for the collection. Defaults to
 title: 'My Favourite Website Layouts'
 ```
 
+#### root <Badge text="added in v1.5.0" type="tip"/>
```

Comment:
should we add a note that this also requires theme support? e.g. users with a custom theme / mandelbrot fork that do not update can't use it.

Reply:
Indeed 👍